assets/css/main.css

```diff
@@ -568,17 +568,60 @@ td.submissions-dropdown {
 .form-inline .form-group div {
     display: inline-block;
 }
-.correct-option {
-  margin-left: -38px;
-  margin-right: 20px;
-  color: green;
+
+/* Mooc-grader questionnaire */
+
+.exercise .aplus-quiz1 .form-group {
+    margin-top: 20px;
+    margin-bottom: 20px;
 }
-.neutral-option {
-  margin-left: -40px;
-  margin-right: 18px;
-  font-size: 18px;
-  color: #f5bc09;
+
+.exercise .aplus-quiz1 .form-control {
+    margin-top: 4px;
+}
+
+.exercise .aplus-quiz1 .option-checked {
+    font-weight: bold;
+}
+
+.exercise .aplus-quiz1 .has-error .form-control {
+    border-width: 2px;
+}
+
+.exercise .aplus-quiz1 .has-error .radio,
+.exercise .aplus-quiz1 .has-error .checkbox {
+    color: #333;
+}
+
+.exercise .aplus-quiz1 .help-block {
+    margin-bottom: 5px;
 }
+
+.exercise .aplus-quiz1 .help-block-checkbox {
+    padding-left: 20px;
+}
+
+.exercise .aplus-quiz1 .has-error .help-block {
+    font-style: bold;
+    margin-bottom: 0px;
+}
+
+.exercise .aplus-quiz1 .glyphicon-ok {
```

Comment:
Could we copy the glyphicon definition here and name this to something else like `.exercise-icon-correct` and another `.exercise-icon-incorrect` or something in those lines. All of these classes are going to be part of the aplus protocol API, so they will stick and we need to keep supporting them in the future, so I would prefer names that require less changing later.